server/datastore/mysql/hosts_test.go

```diff
@@ -1584,7 +1558,9 @@ func TestSaveTonsOfUsers(t *testing.T) {
 				errCh <- err
 				return
 			}
-			atomic.AddInt32(&count1, 1)
+			if atomic.AddInt32(&count1, 1) >= 100 {
```

Comment:
@chiiph let me know if this is still good enough for the goal of the test (which I believe was related to locking when adding many users concurrently). I tried to find a balance between the batch size and the test duration.

Reply:
It's probably fine, the locking happened pretty much right away.